### PR TITLE
fix: checks on session and role-invite api and owner role

### LINKED ISSUE #6125: Buggy checks on session and role-invite and owner role
**Describe the bug**

- If an organizer of an event tries to delete any user who has accepted role invite, then 422 error is produced - `Unprocessable Entity: "Only users can edit their own status"` on FE
This is due to a check on server - 
```
if user:
            if not has_access('is_user_itself', user_id=user.id):
                raise UnprocessableEntity({'source': ''}, "Only users can edit their own status")
```

- When the locked status is tried to be changed by organizer, error is raised.
`KeyError: event_id`
- missing owner from admin_statistics_api
- missing permissions for owner

**Expected behavior**

 - The user should be deleted in first case
 - The session should be updated
 - owner to be added to admin_statistics_api
 - add permissions for owner

**Additional context**

On it

### PR DESCRIPTION
**Pull request by shreyanshdwivedi**
Fixes #6125 

#### Short description of what this resolves:
There were bugs -
 - role deletion (from event overview page)
 - session isLocked updation.
 - missing owner from admin_statistics_api
 - missing permissions for owner

#### Changes proposed in this pull request:
 - Fixed bugs

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The unit tests pass locally with my changes 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

- [ ] All the functions created/modified in this PR contain relevant docstrings.

### CODE REVIEW

**Comment by shreyanshdwivedi** [comment]
@kushthedude  @mrsaicharan1 @uds5501 please review

File: populate_db.py [hunk 1]
```diff
@@ -210,16 +210,21 @@ def create_permissions():
     mod = Role.query.get(4)
     attend = Role.query.get(5)
     regist = Role.query.get(6)
+    ownr = Role.query.get(7)
```
Comment (uds5501) [inline populate_db.py]: is this a typo? @shreyanshdwivedi

Comment (shreyanshdwivedi) [reply populate_db.py]: No @uds5501. Actually I've tried to make the name similar to what we are following for other roles. Like `orgr` for organizer

File: app/api/role_invites.py [hunk 2]
```diff
@@ -112,8 +112,10 @@ def before_update_object(self, role_invite, data, view_kwargs):
         """
         user = User.query.filter_by(email=role_invite.email).first()
         if user:
-            if not has_access('is_user_itself', user_id=user.id):
-                raise UnprocessableEntity({'source': ''}, "Only users can edit their own status")
+            if not has_access('is_organizer', event_id=role_invite.event_id) and not has_access('is_user_itself',
```
Comment (kushthedude) [inline app/api/role_invites.py]: @shreyanshdwivedi Shouldn't `Owner` be also there?

Comment (shreyanshdwivedi) [reply app/api/role_invites.py]: @kushthedude owner has access as an organizer too - 
`app/api/helpers/permission_manager.py`
```py
if user.is_owner(kwargs['event_id']) or user.is_organizer(kwargs['event_id']):
        return view(*view_args, **view_kwargs)
```

**Comment by shreyanshdwivedi** [comment]
@uds5501 @kushthedude @mrsaicharan1 please review :)

File: migrations/versions/43e8c59337ag_.py [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+import sqlalchemy_utils
+
+
+ # revision identifiers, used by Alembic.
+revision = '43e8c59337ag'
+down_revision = '43e8c59337af'
+
+
+def upgrade():
+    op.execute("INSERT INTO permissions(role_id, service_id, can_create, can_read, can_update, can_delete) VALUES((SELECT id FROM roles WHERE name='owner'), 1, true, true, true, true)")
```
Comment (iamareebjamal) [inline migrations/versions/43e8c59337ag_.py]: Why hardcoding service_id

Comment (shreyanshdwivedi) [reply migrations/versions/43e8c59337ag_.py]: @iamareebjamal fixed it. Please have a look

**Pull request merged**